Jenkinsfile

```diff
@@ -245,6 +245,25 @@ pipeline {
                     }
                     post { always { deleteDir() } }
                 }
+                stage('Math functions expressions test') {
+                    agent any
+                    steps {
+                        unstash 'StanSetup'
+                        setupCXX()
+                        script {
+                            dir("lib/stan_math/") {
+                                withEnv(['PATH+TBB=./lib/tbb']) {           
+                                    try { sh "./runTests.py -j${env.PARALLEL} test/expressions" }
+                                    finally { junit 'test/**/*.xml' }
+                                }
+                                withEnv(['PATH+TBB=./lib/tbb']) {           
+                                    sh "python ./test/expressions/test_expression_testing_framework.py"
```

Comment:
@t4c1 can you just verify once more that line 256 and 260 are the correct commands? Other stuff is generic Jenkins stuff c/p from other Jenkins stages.